### DISCUSSION TITLE
requirement.txt

### DISCUSSION THREAD

**Discussion by sorskik** [sherlock-project/sherlock#2233] [Q&A]
I downloaded everything but whenever i try to type python3 -m pip install -r requirements.txt
it tells me 'Defaulting to user installation because normal site-packages is not writeable
ERROR: Could not open requirements file: [Errno 2] No such file or directory: 'requirements.txt'  ' 
what do I have to do ? can someone help me ?

**Comment by ppfeister** [comment]
There is no requirements.txt file in recent versions, and building from source isn't supported.

If you're trying to install, reference the [installation section of the ReadMe][install-readme].

If you're building from source to _make changes to the source code_, then reference the [build from source section][bfs] of the old install guide. Installation from source is __only recommended for contributors__, not for users.

[install-readme]: https://github.com/sherlock-project/sherlock?tab=readme-ov-file#installation
[bfs]: https://github.com/sherlock-project/sherlock/blob/affbf5012ea6f920d50e94ca43479fd2bd470f36/docs/CONTRIBUTING.md
